samples/wifi/scan/src/main.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ * @brief WiFi scan sample
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(scan, CONFIG_LOG_DEFAULT_LEVEL);
+
+#include <nrfx_clock.h>
+#include <zephyr/zephyr.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/init.h>
+
+#include <zephyr/net/net_if.h>
+#include <zephyr/net/wifi_mgmt.h>
+#include <zephyr/net/net_event.h>
+
+#include "net_private.h"
```

Comment:
Why is this header needed?

Reply:
@rlubos , File is included for the declaration of "net_sprint_ll_addr_buf()" function, which is needed for reading BSSID.

Reply:
Ok, I see now, thanks for clarifying.